cnxarchive/utils/ident_hash.py

```diff
@@ -25,6 +27,19 @@ class IdentHashSyntaxError(Exception):
     """Raised when the ident-hash syntax is incorrect."""
 
 
+class IdentHashShortId(Exception):
+    """Raised when the ident-hash id is not a uuid"""
+    def __init__(self, id, version):
+        self.id = id
+        self.version = version
+
+
+class IdentHashMissingVersion(Exception):
```

Comment:
Let's put an IdentHashError base class in place. This way we can except on IdentHashError for all ident-hash problems.